README.md

```diff
@@ -47,4 +47,32 @@ $ npm test
 In the Ginko container, run the application and start the server
 ```
 $ npm start
-```
\ No newline at end of file
+```
+
+# Windows
+On Windows Install the [Docker Toolbox on Windows](https://docs.docker.com/toolbox/toolbox_install_windows/#step-2-install-docker-toolbox)
+, run the tool then open the Docker Quickstart terminal. 
+
+Only the C:\Users directory is linked between the Virtual Machine and windows so place
```

Comment:
Check out if you're able to modify the setting to expose your `D:\` drive as well
https://rominirani.com/docker-on-windows-mounting-host-directories-d96f3f056a2c